test-one-stone-at-a-time.sh

```diff
@@ -48,8 +64,15 @@ run_white_stone () {
     echo "valve3=on"
     sleep .3
     echo "valve3=off"
+    echo "lightbarrier5=on"
+    sleep .45
+    echo "lightbarrier5=off"
 }
 
+echo "Setting up the hardware in 5 secs" >&2
+sleep 5
+echo "Set up the hardware now..." >&2
+start_hardware &
```

Comment:
Backgrounding of start_hardware & stop_hardware is indeed not necessary in current implementation. But it is for run_<color>_stone, as otherwise the functions prevent the toplevel sleeps between the different stone color from starting, which makes it impossible to have multiple stones in the simulation at the same time.
Backgrounding for start_hardware & stop_hardware was just provided, to have a consistent behaviour, if for example sleeps would be added in those two functions.